measurement_kit/pybind/compat-0.3.cpp

```diff
@@ -168,17 +172,18 @@ void RunnerNg::run_unlocked_(Callback<Continuation<>> kickoff) {
     });
 }
 
-void RunnerNg::run_test(Var<NetTest> test, Callback<Var<NetTest>> fn) {
+void RunnerNg::run_test(Var<nettests::BaseTest> test, Callback<Var<nettests::BaseTest>> fn) {
     run([=](Continuation<> complete) {
-        test->begin([=](Error) {
-            // TODO: do not ignore the error
-            test->end([=](Error) {
-                // TODO: do not ignore the error
-                complete([=]() {
-                    fn(test);
-                });
-            });
-        });
+        test->run();
+//        test->begin([=](Error) {
```

Comment:
I guess we're not calling `run_test` anywhere yet, but I'm unsure if what I've done here is ok.